cli/cfncluster/cli.py

```diff
@@ -123,35 +127,60 @@ def main():
+    pstart.add_argument("--config", "-c", dest="config_file", help='specify a alternative config file')
+    pstart.add_argument( "--region", "-r", dest="region", help='specify a specific region to connect to',
+                        default=None)
     pstart.set_defaults(func=start)
 
     pstop = subparsers.add_parser('stop', help='stop the compute-fleet, but leave the MasterServer running for debugging/development')
     pstop.add_argument("cluster_name", type=str, default=None,
                         help='stops the compute-fleet of the provided cluster name.')
+    pstop.add_argument("--config", "-c", dest="config_file", help='specify a alternative config file')
+    pstop.add_argument( "--region", "-r", dest="region", help='specify a specific region to connect to',
+                        default=None)
     pstop.set_defaults(func=stop)
 
     pstatus = subparsers.add_parser('status', help='pull the current status of the cluster')
     pstatus.add_argument("cluster_name", type=str, default=None,
                         help='show the status of cfncluster with the provided name.')
+    pstatus.add_argument("--config", "-c", dest="config_file", help='specify a alternative config file')
+    pstatus.add_argument( "--region", "-r", dest="region", help='specify a specific region to connect to',
+                        default=None)
+    pstatus.add_argument( "--nowait", "-nw", dest="nowait", action='store_true',
+                        help='do not wait for stack events, after executing stack command')
     pstatus.set_defaults(func=status)
 
     plist = subparsers.add_parser('list', help='display a list of stacks associated with cfncluster')
+    plist.add_argument("--config", "-c", dest="config_file", help='specify a alternative config file')
+    plist.add_argument( "--region", "-r", dest="region", help='specify a specific region to connect to',
+                        default=None)
     plist.set_defaults(func=list)
 
     pinstances = subparsers.add_parser('instances', help='display a list of all instances in a cluster')
     pinstances.add_argument("cluster_name", type=str, default=None,
                         help='show the status of cfncluster with the provided name.')
+    pinstances.add_argument("--config", "-c", dest="config_file", help='specify a alternative config file')
+    pinstances.add_argument( "--region", "-r", dest="region", help='specify a specific region to connect to',
+                        default=None)
     pinstances.set_defaults(func=instances)
 
     pconfigure = subparsers.add_parser('configure', help='creating initial cfncluster configuration')
+    pconfigure.add_argument("--config", "-c", dest="config_file", help='specify a alternative config file')
     pconfigure.set_defaults(func=configure)
 
     pversion = subparsers.add_parser('version', help='display version of cfncluster')
+    pversion.add_argument( "--region", "-r", dest="region", help='specify a specific region to connect to',
```

Comment:
Why take a region argument for "cfncluster version"?

Reply:
removed.